### DISCUSSION TITLE
How can i enable the download option?

### DISCUSSION THREAD

**Discussion by thejanz** [jorisdugue/h5p-bundle#33] [Q&A] [answered]
Hey, how can i enable the download option? :)

**Comment by Zaruike** [comment, marked as answer]
> Hey, how can i enable the download option? :)
Hello, if you have enabled the permission you need to have the role **ROLE_H5P_DOWNLOAD_ALL** and **ROLE_H5P_COPY_ALL**
Or if you have disabled the permission you just need set the export at 3 in configuration file
